app/partials/project-card.cjsx

```diff
@@ -23,10 +30,10 @@ module.exports = React.createClass
       'data-no-avatar': true unless @props.project.avatar
 
     <Link {...linkProps} className="project-card">
-      <svg className="project-card-space-maker" viewBox="0 0 2 1" width="100%"></svg>
+      <svg className="project-card-space-maker" viewBox="0 0 2 150px" width="100%"></svg>
       <div className="details">
         <div className="name">{@props.project.display_name}</div>
         <div className="owner">{owner?.display_name ? 'LOADING'}</div>
-        <div className="description">{@props.project.description}</div>
+        <button type="button" className="ghost-button project-card-button"><Translate content="projectsCard.button" /></button>
```

Comment:
Yeah I think making it look like a button is fine. Nesting focusables was my concern. tabindex="-1" should be fine. I'll merge this in a few minutes if that's cool with you.